examples/fuse_examples/imaging/classification/mnist/run_mnist.py

```diff
@@ -245,7 +251,7 @@ def run_infer(paths: dict, infer_common_params: dict):
 
     # convert list of batch outputs into a dataframe
     infer_df = convert_predictions_to_dataframe(predictions)
-    save_dataframe(infer_df, infer_common_params['infer_filename'])
+    save_dataframe(infer_df, infer_file)
     
 
 ######################################
```

Comment:
should be Eval Common Params

Reply:
Already fixed it and missed a push, thanks